### PR TITLE
[SYCL] Update README-sycl.md for Chapter "Recommended release" and "News"

### PR DESCRIPTION
**Pull request by arthw**
- Self Reported Review Complexity:
    - [X] Review Complexity : Low
    - [ ] Review Complexity : Medium
    - [ ] Review Complexity : High
- [X] I have read the [contributing guidelines](https://github.com/ggerganov/llama.cpp/blob/master/CONTRIBUTING.md)

1. Add chapter "Recommended Release"
2. Update the "News"

### CODE REVIEW

File: README-sycl.md [hunk 1]
```diff
@@ -31,8 +32,23 @@ When targeting **Intel CPU**, it is recommended to use llama.cpp for [Intel oneM
...
 
+## Recommended Release
+
+The SYCL backend would be broken by some PRs due to no online CI.
+
+Here is the releases with good quality:
+
+|Commit ID|Tag|Release|
+|-|-|-|
+|fb76ec31a9914b7761c1727303ab30380fd4f05c|b3038 |[llama-b3038-bin-win-sycl-x64.zip](https://github.com/ggerganov/llama.cpp/releases/download/b3038/llama-b3038-bin-win-sycl-x64.zip) |
```
Comment (airMeng) [inline README-sycl.md:43]: @luoyu-intel effectiveness on windows?

Comment (arthw) [reply README-sycl.md:43]: Yes, I test the win package and result is correct:
  Download the zip file and decompress
  Run the tools directly.

File: README-sycl.md [hunk 2]
```diff
@@ -31,8 +32,23 @@ When targeting **Intel CPU**, it is recommended to use llama.cpp for [Intel oneM
 
 It has the similar design of other llama.cpp BLAS-based paths such as *OpenBLAS, cuBLAS, etc..*. In beginning work, the oneAPI's [SYCLomatic](https://github.com/oneapi-src/SYCLomatic) open-source migration tool (Commercial release [Intel® DPC++ Compatibility Tool](https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compatibility-tool.html)) was used for this purpose.
 
+## Recommended Release
+
+The SYCL backend would be broken by some PRs due to no online CI.
+
+Here is the releases with good quality:
```
Comment (airMeng) [inline README-sycl.md:39]: some more precise description

```suggestion
The following release is verified on GPU XXX/XXX/XXX with OneAPI xxx, Windows 11 xxx:
```

Comment (arthw) [reply README-sycl.md]: Yes, add the hw/os/oneapi info in new column.

File: README-sycl.md [hunk 3]
```diff
@@ -31,8 +32,23 @@ When targeting **Intel CPU**, it is recommended to use llama.cpp for [Intel oneM
 
 It has the similar design of other llama.cpp BLAS-based paths such as *OpenBLAS, cuBLAS, etc..*. In beginning work, the oneAPI's [SYCLomatic](https://github.com/oneapi-src/SYCLomatic) open-source migration tool (Commercial release [Intel® DPC++ Compatibility Tool](https://www.intel.com/content/www/us/en/developer/tools/oneapi/dpc-compatibility-tool.html)) was used for this purpose.
 
+## Recommended Release
+
+The SYCL backend would be broken by some PRs due to no online CI.
+
+The following release is verified with good quality:
+
+|Commit ID|Tag|Release|Verify Platform|
```
Comment (airMeng) [inline README-sycl.md:41]:
```suggestion
|Commit ID|Tag|Release|Verified Platform|
```

**Review by airMeng — APPROVED** [review]

**Pull request merged**